demos/react-jsnation/package.json

```diff
@@ -50,6 +51,8 @@
   "scripts": {
     "test:lint": "eslint",
     "test:build": "tsc -b",
-    "test:types": "tsc -b"
+    "test:specs": "vitest --dir ./tests --run",
+    "test:types": "tsc -b",
+    "ui:test": "vitest --dir ./tests --ui"
```

Comment:
Named this way to avoid triggering it as part of the workspace-wide test run that calls all `test:*` scripts. If there is a better naming pattern you'd prefer, I'm happy to go with that.